cmd/minishift/cmd/start_preflight.go

```diff
@@ -55,6 +55,11 @@ func preflightChecksBeforeStartingHost() {
 
 	proxy := os.Getenv("HTTPS_PROXY")
 
+	// If --skip-preflight-check flag is set to true means do not check
+	if viper.GetBool(configCmd.SkipPreflightChecks.Name) {
+		return
+	}
+
```

Comment:
It should be good if this also comes at the top, just after function start, we don't need to assign those variable if we are not using it in case of skip.

Reply:
Agree here @praveenkumar .

Reply:
Duplication. Change to:

```
   if  shouldPreflightChecksBeSkipped() {
      return
   }
```